tools/csv-merger/csv-merger.go

```diff
@@ -118,6 +118,7 @@ var (
 	crdDir                        = flag.String("crds-dir", "", "the directory containing the CRDs for apigroup validation. The validation will be performed if and only if the value is non-empty.")
 	hcoKvIoVersion                = flag.String("hco-kv-io-version", "", "KubeVirt version")
 	kubevirtVersion               = flag.String("kubevirt-version", "", "Kubevirt operator version")
+	kvVirtLauncherOSVersion       = flag.String("kubevirt-virt-launcher-os-version", "", "Kubevirt Virt launcher OS version")
```

Comment:
Thaks for the change, but the command line parameter name is still too long. please drop `kubevirt-`